ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -0,0 +1,193 @@
+      image: ${LOCAL_IMAGE_BIN}
+      resources:
+        requests:
+          cpu: 1
+          memory: 300Mi
+        limits:
+          cpu: 3
+          memory: 4Gi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      env:
+      - name: HOME
+        value: /tmp/home
+      - name: TEST_COMMAND
+        value: "${TEST_COMMAND}"
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        trap 'touch /tmp/shared/exit' EXIT
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        cp "$(which oc)" /tmp/shared/
+
+        mkdir -p "${HOME}"
+
+        # wait until the setup job creates admin.kubeconfig
+        while true; do
+          if [[ ! -f  /tmp/shared/_data/_out/admin.kubeconfig ]]; then
+            sleep 15 & wait
+            continue
+          fi
+          break
+        done
+        echo "Found shared kubeconfig"
+
+        # don't let clients impact the global kubeconfig
+        export HOME=/tmp/shared
+        cp -r /tmp/shared/_data /go/src/github.com/openshift/openshift-azure/
+        export KUBECONFIG=/go/src/github.com/openshift/openshift-azure/_data/_out/admin.kubeconfig
+
+        PATH=/usr/libexec/origin:$PATH
+
+        # create the environment, but exit before we run tests
+        #TODO: RUN TESTS HERE
+        oc get pods --all-namespaces 
+        exit 0
+
+    # Runs an install
+    - name: setup
+      image: ${LOCAL_IMAGE_BIN}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-secrets-azure
+        mountPath: /etc/azure/credentials
+      env:
+      - name: INSTANCE_PREFIX
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      - name: TYPE
+        value: ${CLUSTER_TYPE}
+      command:
+      - /bin/bash
+      - -c
+      - |
+        # Test suite specific configuration.
+        export HOME=/tmp/shared
+        source /etc/azure/credentials/secret
+        az login --service-principal -u ${AZURE_CLIENT_ID} -p ${AZURE_CLIENT_SECRET} --tenant ${AZURE_TENANT_ID} &>/dev/null
+        # TODO: Configure these from prow
+        export DNS_DOMAIN=osadev.cloud
+        export DNS_RESOURCEGROUP=dns
+        export DEPLOY_VERSION=v3.10
+        export RUN_SYNC_LOCAL=true
+        export DEPLOY_OS="centos7"
+        export IMAGE_RESOURCEGROUP=images
+        # Azure cli fails miserably now with az image list 
+        # https://github.com/Azure/azure-cli/issues/7019
+        #export IMAGE_RESOURCENAME=$(az image list -g $IMAGE_RESOURCEGROUP -o json --query "[?starts_with(name, '${DEPLOY_OS:-rhel7}-${DEPLOY_VERSION//v}') && tags.valid=='true'].name | sort(@) | [-1]" | tr -d '"')
+        export IMAGE_RESOURCENAME=centos7-3.10-201808031447
+        # TODO: for full e2e image needs to be built too  
+        cd /go/src/github.com/openshift/openshift-azure/
+       
+        ./hack/create.sh ${INSTANCE_PREFIX}
+
+        # if completed pass generated data to other stage
```

Comment:
You probably need `set -e` to run this only when ./hack/create.sh completes successfully.

Reply:
took same pattern as other templates `set -euo pipefail`